x-pack/test/functional/es_archives/security_solution/timestamp_override_6/mappings.json

```diff
@@ -20,7 +20,6 @@
     },
     "settings": {
       "index": {
-        "refresh_interval": "1s",
```

Comment:
Just curious, why did we remove `refresh_interval`?

Reply:
Sure! The `refresh_interval` is not a valid config in Serverless, and We found out that we don't even need for ESS either since out tests don't valid against that fields